contracts/contracts/strategies/AaveStrategy.sol

```diff
@@ -70,7 +108,12 @@ contract AaveStrategy is InitializableAbstractStrategy {
 
         IAaveAToken aToken = _getATokenFor(_asset);
         emit Withdrawal(_asset, address(aToken), _amount);
-        aToken.redeem(_amount);
+        uint256 actual = _getLendingPool().withdraw(
+            _asset,
+            _amount,
+            address(this)
+        );
+        require(actual >= _amount, "Did not withdraw enough");
```

Comment:
I guess this was just a cheap sanity check.  Do you think it would be better to do `actual == amount`?

Reply:
IMO, I think that would be more appropriate since `actual` should never be greater than `_amount`.  I was also suggesting the revert message wasn't accurate.